src/Language/Haskell/Ghcid/Types.hs

```diff
@@ -12,8 +12,11 @@ import Data.Data
 import Control.Exception.Base (Exception)
 
 -- | GHCi shut down
-data GhciError = UnexpectedExit String String
-    deriving (Show,Eq,Ord,Typeable,Data)
+data GhciError = UnexpectedExit {
+    _ghciErrorCmd :: String
```

Comment:
Why the leading underscores? These types have no lenses (which is the common reason for doing that), but I'd rather they were just unadorned non-underscore. I'll merge and fix up after.